src/views/ModelPlan/TaskList/Payment/ClaimsBasedPayment/index.tsx

```diff
@@ -0,0 +1,577 @@
+        data-testid="model-plan-name"
+      >
+        <Trans i18nKey="modelPlanTaskList:subheading">
+          indexZero {modelName} indexTwo
+        </Trans>
+      </p>
+      <p className="margin-bottom-2 font-body-md line-height-sans-4">
+        {h('helpText')}
+      </p>
+
+      <AskAQuestion modelID={modelID} />
+
+      <Formik
+        initialValues={initialValues}
+        onSubmit={values => {
+          handleFormSubmit(values, 'next');
+        }}
+        enableReinitialize
+        innerRef={formikRef}
+      >
+        {(formikProps: FormikProps<ClaimsBasedPaymentFormType>) => {
+          const {
+            errors,
+            handleSubmit,
+            setFieldValue,
+            setErrors,
+            values
+          } = formikProps;
+          const flatErrors = flattenErrors(errors);
+          return (
+            <>
+              {Object.keys(errors).length > 0 && (
+                <ErrorAlert
+                  testId="formik-validation-errors"
+                  classNames="margin-top-3"
+                  heading={h('checkAndFix')}
+                >
+                  {Object.keys(flatErrors).map(key => {
+                    return (
+                      <ErrorAlertMessage
+                        key={`Error.${key}`}
+                        errorKey={key}
+                        message={flatErrors[key]}
+                      />
+                    );
+                  })}
+                </ErrorAlert>
+              )}
+              <GridContainer className="padding-left-0 padding-right-0">
+                <Grid row gap>
+                  <Grid desktop={{ col: 6 }}>
+                    <Form
+                      className="margin-top-6"
+                      data-testid="payment-claims-based-payment-form"
+                      onSubmit={e => {
+                        handleSubmit(e);
+                      }}
+                    >
+                      <PageHeading
+                        headingLevel="h3"
+                        className="margin-bottom-3"
+                      >
+                        {t('claimSpecificQuestions')}
+                      </PageHeading>
+
+                      <FieldGroup
+                        scrollElement="payClaims"
+                        error={!!flatErrors.payClaims}
+                        className="margin-top-4"
+                      >
+                        <Label htmlFor="payment-pay-claims">
+                          {t('selectClaims')}
+                        </Label>
+                        <p className="text-base margin-bottom-1 margin-top-05">
+                          {t('selectClaimsSubcopy')}
+                        </p>
+                        <FieldErrorMsg>{flatErrors.payClaims}</FieldErrorMsg>
+
+                        <Field
+                          as={MultiSelect}
+                          id="payment-pay-claims"
+                          name="beneficiaries"
```

Comment:
For all Formik components "name" is not arbitrary, it should always reference the field name for the formikValue